api/v2alpha1/go/pipelinespec/pipeline_spec.pb.go

```diff
@@ -1,17 +1,16 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.25.0
-// 	protoc        v3.17.3
+// 	protoc-gen-go v1.27.1
+// 	protoc        v3.6.1
```

Comment:
@capri-xiyue why did you need a new version of protoc?

I installed protoc from `sudo apt install protobuf-compiler`, it's a little old, but convenient to install. Can I keep using the old version?

Reply:
If the old version support protocol reflect, it should be ok. I use new versions because I need the protocol reflect method so that I can use protocol json to do marshal.

Reply:
Rethinking over this, I think we should use gcr.io/ml-pipeline-test/api-generator directly, so that we can ensure all versions match despite developer.